src/main/java/com/spotify/docker/client/messages/HostConfig.java

```diff
@@ -1036,7 +1038,12 @@ public String toString() {
       } else if (from == null || from.equals("")) {
         return to;
       } else if (readOnly == null || !readOnly) {
-        return from + ":" + to;
+          if (noCopy == null || !noCopy) {
```

Comment:
So probably, 'ro' has to do with the operation and 'nocopy' with the initialization...I see